fontforgeexe/cvfreehand.c

```diff
@@ -440,8 +440,6 @@ return( ((p-si->pressure1)*si->radius2 + (si->pressure2-p)*(si->width/2))/
 static void TraceFigureCPs(SplinePoint *last,SplinePoint *cur,TraceData *tlast,TraceData *tcur) {
     TraceData *pt;
 
-    last->nonextcp = false; cur->noprevcp=false;
```

Comment:
Flag: These are immediately checked after calling this function, pretty sure these need to be correctly set by the return of this function

Reply:
Fixed at the caller.